docs/specs/code/MULTI_CURRENCY.md

```diff
@@ -0,0 +1,55 @@
+# Abstract
+
+Describe the ability to see and input ADA amount in another currency.
+
+# Proposal
+
+This feature includes two aspects:
+
+## 1. Input
+When the UI contains an ADA amount input, it should let the user input the number in another currency and covert it to ADA amount at the present conversion rate.
```

Comment:
we should define and sign on which currencies we will support

Reply:
Added.